libraries/precisoUtils/bidUtils.js

```diff
@@ -11,23 +12,7 @@ export const buildRequests = (endpoint) => (validBidRequests = [], bidderRequest
     // bidRequest: bidderRequest,
     id: validBidRequests[0].auctionId,
     cur: validBidRequests[0].params.currency || ['USD'],
```

Comment:
You need to first read currency out of config.getConfig(‘currency.adServerCurrency’). Your local params can be an alternate source.